client/src/components/pages/RobotOverview/RobotOverview.jsx

```diff
@@ -95,6 +95,17 @@ const RobotOverview = () => {
       });
   };
 
+  /**
+   * @description Removes the robot with the given id from the overview
+   * @param {String} robotId Id of the robot to be removed
+   */
+  const removeRobotFromOverview = (robotId) => {
+        const newRobotList = robotList.filter(foundRobotId => foundRobotId !== robotId)
+        setRobotList([]);
+        setRobotList(newRobotList);
+        retrieveBotList(userId);
```

Comment:
Why is this still called bot instead of robot? Additionally, why are rows 104 and 105 needed when they are almost exactly called the same way in retrieveBotList() ?